tasks/test_results_processor.py

```diff
@@ -57,9 +57,16 @@ def __init__(self, err_msg, file_content, parser="", parser_err_msg=""):
 class ParserNotSupportedError(Exception): ...
 
 
-def get_existing_tests(db_session: Session, repoid: int) -> dict[str, Test]:
-    existing_tests = db_session.query(Test).filter(Test.repoid == repoid).all()
-    return {test.id_: test for test in existing_tests}
+def get_existing_flag_bridges(
+    db_session: Session, repoid: int, flags: list[str]
```

Comment:
the `flags` argument is unused. what is its purpose?

Reply:
fixed this